types/chrome/index.d.ts

```diff
@@ -1538,7 +1538,7 @@ declare namespace chrome.contextMenus {
          * Since Chrome 35.
          * The parent ID, if any, for the item clicked.
          */
-        parentMenuItemId?: any;
+        parentMenuItemId?: number | string | undefined;
```

Comment:
you don't have to include undefined, since that's automatically added for optional values